api/producer/updateDocumentReference/src/v1/handler.py

```diff
@@ -77,13 +77,36 @@ def document_pointer_exists(
     )
 
 
+def _sort_key(key):
+
+    if type(key) == list:
+        return sorted(key)
+    elif type(key) == dict:
+        key_values = list(key.items())[0][1]
+
+        if type(key_values) == dict:
+            return dict(sorted(key_values.items()))
+        elif type(key_values) == list:
+            keys_list = []
+            for sub_keys in key_values:
+                keys_list.extend(list(sub_keys.keys()))
+
+            return sorted(key_values, key=lambda item: sorted(keys_list)[0])
```

Comment:
would the following not be sufficient (would sort any dictionary by keys then sort all values):

```suggestion
        return {k: _sort_key(key[k]) for k in sorted(key.keys())}
```